include/helib/CModulus.h

```diff
@@ -123,6 +123,9 @@ class Cmodulus
   void FFT(NTL::vec_long& y, const NTL::ZZX& x) const;
   // y = FFT(x)
   void FFT(NTL::vec_long& y, const zzX& x) const;
+  // y = FFT(x)
+  void FFT(NTL::vec_long& y, NTL::zz_pX& x) const;
```

Comment:
Should the second arg not be const ref?

Reply:
FFT_aux (https://github.com/helibproject/HElib/blob/master/include/helib/CModulus.h#L81) requires a non-const argument. This could probably be fixed with an extra copy to the BluesteinFFT call (https://github.com/helibproject/HElib/blob/master/src/CModulus.cpp#L435)